src/components/cleanButton.jsx

```diff
@@ -0,0 +1,5 @@
+export const CleanButton = () => {
```

Comment:
here you should pass "props"
```suggestion
export const CleanButton = ({cleanFields}) => {
```

Reply:
Далее тебе нужно соответственно передать эту функцию в onclick кнопки ниже